subsys/bootloader/CMakeLists.txt

```diff
@@ -171,19 +171,92 @@ include (${CMAKE_CURRENT_LIST_DIR}/cmake/debug_keys.cmake)
 
 set(sign_depends kernel_elf ${key_file_depends})
 
+set(hash_file firmware.sha256)
```

Comment:
What do you think about creating `subsys/bootloader/cmake/sign.cmake` and having an `include(${CMAKE_CURRET_LIST_DIR}/sign.cmake` ?

Reply:
Sounds good, will change.